src/main/java/org/spongepowered/common/mixin/core/server/MixinDedicatedServer.java

```diff
@@ -55,4 +58,18 @@ public void setGuiEnabled() {
         this.guiIsEnabled = false;
     }
 
+    /**
+     * @author ryantheleach
+     *
+     * Purpose: This disables the new check in minecraft 1.8 that checks the OP list to check if there is no operators
+     * Reasoning: There has been complaints about 1.8 breaking spawn protection see: https://github.com/SpongePowered/SpongeCommon/issues/15
+     * UserList.hasEntries() is named incorrectly in MCP, it should be isEmpty
```

Comment:
You might want to report this to [MCPBot-Issues](https://github.com/ModCoderPack/MCPBot-Issues/issues).